src/riak_cs_console.erl

```diff
@@ -62,6 +63,86 @@ cluster_info([OutFile]) ->
             error
     end.
 
+%% @doc Audit bucket ownership by comparing information between
+%%      users bucket and buckets bucket.
+audit_bucket_ownership(_Args) when is_list(_Args) ->
+    audit_bucket_ownership().
+
+audit_bucket_ownership() ->
+    {ok, RcPid} = riak_cs_riak_client:start_link([]),
+    try
+        Inconsistencies = audit_bucket_ownership0(RcPid),
+        io:format("~p", [Inconsistencies])
```

Comment:
`~n` would make this look better. Also, a message like "No inconsistency found." will be more informative when `Inconsistencies` is just `[]`.